src/Compilers/CSharp/Portable/Symbols/Symbol_Attributes.cs

```diff
@@ -345,7 +347,10 @@ internal virtual void PostDecodeWellKnownAttributes(ImmutableArray<CSharpAttribu
             if (lazyCustomAttributesBag.SetAttributes(boundAttributes))
             {
                 this.RecordPresenceOfBadAttributes(boundAttributes);
```

Comment:
>this.RecordPresenceOfBadAttributes(boundAttributes); [](start = 16, length = 52)

Should this line be moved into the ```if``` too?

Reply:
I think it would be ok to leave because it just records a bit in the module. But for clarity (no side-effects when using `attributeMatchesOpt` filter) I'll move this into the `if`.

---
In reply to: [156449683](https://github.com/dotnet/roslyn/pull/23647#discussion_r156449683) [](ancestors = 156449683)